src/tv2-common/__tests__/testUtil.ts

```diff
@@ -0,0 +1,3 @@
+export function prefixLayer(layerName: string) {
```

Comment:
> Also, I think this might be overkill to create an entire utils file for.

Where else would you put it? Cross-importing from test files causes some issues that I want to avoid